core/Core.rdf

```diff
@@ -329,7 +354,7 @@
 		<iof-av:adaptedFrom>https://en.wikipedia.org/wiki/Business_purpose</iof-av:adaptedFrom>
 		<iof-av:counterExample>any function of a non-profit organization</iof-av:counterExample>
 		<iof-av:firstOrderLogicAxiom>LA1: BusinessFunction(x) → Function(x) ∧ ∃o,∃i(Organization(o) ∧ ObjectiveSpecification(i) ∧ functionOf(x,o) ∧ genericallyDependsOnAtSomeTime(i,o) ∧ prescribedBy(x,i)) ∧ ∀y(hasRealization(x,y) → BusinessProcess(y))</iof-av:firstOrderLogicAxiom>
-		<iof-av:firstOrderLogicAxiom>LA2: Function(x) ∧ ∃o,∃i,∃p(Organization(o) ∧ ObjectiveSpecification(i) ∧ BusinessProcess(p) ∧ functionOf(x,o) ∧ genericallyDependsOnAtSomeTime(i,o) ∧ prescribedBy(x,i)) ∧ hasRealization(x,p)) → BusinessFunction(x)</iof-av:firstOrderLogicAxiom>
+		<iof-av:firstOrderLogicAxiom>LA2: Function(x) ∧ ∃o,∃i,∃p(Organization(o) ∧ ObjectiveSpecification(i) ∧ BusinessProcess(p) ∧ functionOf(x,o) ∧ genericallyDependsOnAtSomeTime(i,o) ∧ prescribedBy(x,i) ∧ hasRealization(x,p)) → BusinessFunction(x)</iof-av:firstOrderLogicAxiom>
```

Comment:
"∃o,∃i,∃p" - see my other comments on multiple quantifiers.

Reply:
all the quantifiers should be "fixed" now, please let me know if any remains in the Core file